src/Common/Resources/Clockify.ClockifyMessageSource.resx

```diff
@@ -154,4 +154,7 @@
         <value>omg that's a looot of time😰{0}
 Please don't exceed one year window</value>
     </data>
+    <data name="FollowUp" xml:space="preserve">
+        <value>Hey! It looks like you haven't set up a clockify token already</value>
```

Comment:
@simone3991 do you like this message or do you have any suggestions?

Reply:
Do this:

Hey 👋 I noticed you never setup a Clockify token...
Help me help you! Once you're setup I will assist you in your daily time tracking.